script/optimization_comparison.py

```diff
@@ -65,16 +65,18 @@ def optimization_paths():
     path = {}
 
     fig, axes = plt.subplots(4, 3)
+    fig.tight_layout(h_pad=1.0)
+    fig.subplots_adjust(top = 0.9)
     fig.delaxes(axes[0, 0])
     fig.delaxes(axes[0, 2])
-    fig.suptitle('Comparison of Optimation Methods on Rosenbrock Function')
+    fig.suptitle('Comparison of Optimation Methods on Rosenbrock Function (top: optimization function)')
```

Comment:
@hdashnow it appears to have been called "cost function" before (at least in my build). I think that's probably the correct terminology. =)